lib/install.js

```diff
@@ -242,8 +242,15 @@ Installer.prototype.run = function (_cb) {
     }
   }
 
+  var preinstallSteps = []
   var installSteps = []
   var postInstallSteps = []
+
+  preinstallSteps.push(
```

Comment:
@iarna A new phase is optional and I'm fine removing it. Before getting this to work I had a few other steps there and it made some sense. Removing now and pushing a few tweaks!